packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -542,16 +549,17 @@ export async function DocsQA<Doc extends Document>(props: DocsQAProps<Doc>) {
   if (status !== Corpus.LoadingState.COMPLETED) {
     return `Corpus is not loaded. It's in state: ${status.toString()}`;
   }
-  const docs = await props.corpus.search(props.question);
+  const docs = await props.corpus.search(props.question, { limit: props.limit });
   return (
     <ChatCompletion>
       <SystemMessage>
-        You are a customer service agent. Answer questions truthfully. Here is what you know:
+        You are a trained question answerer. Answer questions truthfully, using only the document excerpts below. Do not
+        use any other knowledge you have about the world. If you don't know how to answer the question, just say "I
+        don't know." Here are the relevant document excerpts you have been given:
         {docs.map((doc) => (
-          // TODO improve types
-          // @ts-expect-error
           <props.docComponent doc={doc} />
         ))}
+        And here is the question you must answer:
```

Comment:
I think this might be unnecessary in a chat model but 🤷

Reply:
yeah, I was looking for a citation here but couldn't find one. I'll file a TODO to check the OAI cookbook on this.